extension/src/ctp.js

```diff
@@ -1,17 +1,14 @@
-const fetch = require('node-fetch')
-const { merge } = require('lodash')
-const { createClient } = require('@commercetools/sdk-client')
-const {
-  createAuthMiddlewareForClientCredentialsFlow,
-} = require('@commercetools/sdk-middleware-auth')
-const {
-  createUserAgentMiddleware,
-} = require('@commercetools/sdk-middleware-user-agent')
-const { createHttpMiddleware } = require('@commercetools/sdk-middleware-http')
-const { createQueueMiddleware } = require('@commercetools/sdk-middleware-queue')
-const { createRequestBuilder } = require('@commercetools/api-request-builder')
-const packageJson = require('../package.json')
-
+import fetch from 'node-fetch'
+import lodash from 'lodash'
+import { createClient } from '@commercetools/sdk-client'
+import { createAuthMiddlewareForClientCredentialsFlow } from '@commercetools/sdk-middleware-auth'
+import { createUserAgentMiddleware } from '@commercetools/sdk-middleware-user-agent'
+import { createHttpMiddleware } from '@commercetools/sdk-middleware-http'
+import { createQueueMiddleware } from '@commercetools/sdk-middleware-queue'
+import { createRequestBuilder } from '@commercetools/api-request-builder'
+import packageJson from '../package.json'
```

Comment:
Does this work? I thought it's not in node 16 or only behind an experimental flag: https://github.com/tc39/proposal-json-modules. We did this instead: https://github.com/commercetools/commercetools-subscriptions/blob/main/src/utils/utils.js#L6

Reply:
Yes, you are right. I tried implementing similar to the link you provided.
But I am getting below error: 

`2:30  error  Parse errors in imported module '../../src/ctp.js': Cannot use keyword 'await' outside an async function`

How did you make it work for the below code? 
https://github.com/commercetools/commercetools-subscriptions/blob/2a778859c04d6455ae8e6ccbf295ec7edac5813e/src/index.js#L5

Reply:
This is called top-level await and it's available since node 14.8: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/await#top_level_await. It should work here too since we're using nodejs > 16.

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/929/commits/7e322331ee0edf5626d813fbfc5f8be0722a4fea

And also had to do some additional fixes. Please check the latest commits.